=

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import Image from 'next/image'
+// MUI //
+import Drawer from '@mui/material/Drawer'
+import { Box } from '@mui/system'
+import Collapse from '@mui/material/Collapse'
+// other mui dependencies
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+
+// icons and text
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import StarIcon from '@mui/icons-material/Star'
+import SettingsIcon from '@mui/icons-material/Settings'
+
+export default function NavigationDrawer() {
+  // Params for the Drawer
+  const drawerWidth = 280
+  // Logo definition
+  const logo = (
+    <Image
+      src="/../public/images/CCAHT-Logo.png"
+      alt="CCAHT-logo"
+      height="70"
+      width="239"
```

Comment:
add a prop to the `Image` defined like:
```
style={{ objectFit: 'contain' }}
```

this will use the `object-fit` CSS property to ensure the image has the correct aspect ratio.